LibraBFT/Impl/OBM/Util.agda

```diff
@@ -0,0 +1,27 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
+
+   Copyright (c) 2021, Oracle and/or its affiliates.
+   Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
+-}
+
+open import LibraBFT.Impl.OBM.Rust.RustTypes
+open import LibraBFT.ImplShared.Consensus.Types
+open import LibraBFT.ImplShared.Consensus.Types.EpochIndep
+open import LibraBFT.Prelude
+
+module LibraBFT.Impl.OBM.Util where
+
+-- to tolerate f failures, cluster must contain at least n ≥ 3f + 1 nodes,
+-- where n − f nodes form a quorum
+
+numNodesNeededForNFailures : U64 -> U64
```

Comment:
Just noting that we now have `EpochConfig` construction general enough to allow non-uniform voting power now!  No change requested, but maybe we could have a comment to that effect wherever makes sense.

Reply:
Comment added.